src/schemas/base-state-machine.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Could you update the `state-machine.json` schema so it extends this base to avoid duplication?
If I understand well, it is the same without `Version` and `TimeoutSeconds` properties?
Also, I think we could use this base in the `Branches` properties of the `parallel.json` schema instead of using  `state-machine.json`?

Reply:
I recall trying this and hitting issues with additionalProperties. I think it's possible to get right with JSON Schema + AJV, I just didn't have the time to work through the issue. My approach was to have the base as its own file and then have `state-machine.json` reference it with an $allOf and then add its own properties.